tg_bot/modules/welcome.py

```diff
@@ -140,6 +142,23 @@ def new_member(bot: Bot, update: Update):
             if sent:
                 sql.set_clean_welcome(chat.id, sent.message_id)
 
+        mute_user = sql.get_mute_pref(chat.id)
+        if mute_user:
+            try:
+                bot.restrict_chat_member(chat.id, new_mem.id,
+                                     can_send_messages=False,
+                                     can_send_media_messages=False,
+                                     can_send_other_messages=False,
+                                     can_add_web_page_previews=False)
+            except TelegramError:
+                pass
+
+        return "<b>{}:</b>" \
+           "\n#NEW_USER" \
```

Comment:
Please don't try to add multiple features in on PR, especially if they haven't been discussed, or even mentioned.